samples/nextjs/data/routes/en.yml

```diff
@@ -52,6 +52,7 @@ placeholders:
             <p>To remove all of the default sample content (the Styleguide and GraphQL routes) and start out with an empty JSS app:</p>
             <ol>
               <li>Delete <code>/src/components/Styleguide*</code> and <code>/src/components/GraphQL*</code></li>
```

Comment:
change
`<li>Delete <code>/src/components/Styleguide*</code> and <code>/src/components/GraphQL*</code></li>`
to 
`<li>Delete <code>/src/components/*</li>`